index.html

```diff
@@ -4,7 +4,7 @@
   <title>Javascript Tetris</title>
   <style>
     body      { font-family: Helvetica, sans-serif; }
-    #tetris   { margin: 1em auto; padding: 1em; border: 4px solid black; border-radius: 10px; background-color: #F8F8F8; }
+    #tetris   { margin: 1em auto; padding: 1em; border: 4px solid black; border-radius: 10px; background-color: #F8F8F8; z-index: 1000 }
```

Comment:
```suggestion
    #tetris   { margin: 1em auto; padding: 1em; border: 4px solid black; border-radius: 10px; background-color: #F8F8F8; z-index: 40 }
```
Lets not make the zIndex to high